src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterExpressionTestCases/OpMatrix_Eq_Float_V1CompatDisabled_Overrides.txt

```diff
@@ -0,0 +1,6 @@
+#override: OpMatrix_Eq_Float_V1CompatDisabled.txt
+#SETUP: disable:PowerFxV1CompatibilityRules
+
+// Bug in pre-V1, fixed in V1
```

Comment:
I'm not sure why we need to differentiate pre-V1?  Canvas would have PowerFxV1Compat turned off, don't the #SETUP directives take care of this?

Reply:
Ah, got it.  Thanks!